cloudant-client/src/main/java/com/cloudant/client/api/ClientBuilder.java

```diff
@@ -623,13 +623,17 @@ public ClientBuilder readTimeout(long readTimeout, TimeUnit readTimeoutUnit) {
      * service instance.
```

Comment:
This javadoc needs updating since it no longer inspects the environment variable.